doc/dune-files.rst

```diff
@@ -877,6 +877,30 @@ instead of this stanza. For example:
 
   (dirs :standard \ <sub-dir1> <sub-dir2> ...)
 
+.. _dune-vendored_dirs:
+
+vendored_dirs (since 1.6)
```

Comment:
1.6?

Reply:
woops, copy pasted and forgot to update!